### ISSUE TITLE
Next Week Plan (03-26-21)

### ISSUE THREAD

**Issue by bagmk** [bagmk/Quantum_Machine_Learning_Express#5]
## Plan for next week

- ~Try for circuit 2 for two qubit~
- ~Presentation (main work this week)~
- ~Split the function into quantum circuit and loss function~
- ~Try for 100 data with less fluctuation.~
- Understand the interface between Qiskit and PyTorch (continued)

## Meeting summary

- Discuss the plan for the presentation
- Discuss the neural network and parametric quantum circuit
- Hyperparameters and parameters
- Pytorch
